README.md

```diff
@@ -1,6 +1,6 @@
 # Instaclustr SSTable tools
 
-[![Maven Badge](https://img.shields.io/maven-central/v/com.instaclustr/ic-sstable-tools-4.0.0.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.instaclustr%22%20AND%20a:%22ic-sstable-tools-4.0.0%22)
+[![Maven Badge](https://img.shields.io/maven-central/v/com.instaclustr/ic-sstable-tools-4.0.1.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.instaclustr%22%20AND%20a:%22ic-sstable-tools-4.0.1%22)
```

Comment:
This currently leaves a not found badge, I assume this will change after it is merged in?

Reply:
It will need to be released into maven central